src/studio-home/factories/mockApiResponses.jsx

```diff
@@ -46,51 +49,58 @@ export const generateGetStudioHomeDataApiResponse = () => ({
 });
 
 export const generateGetStudioCoursesApiResponse = () => ({
-  archivedCourses: [
-    {
-      courseKey: 'course-v1:MachineLearning+123+2023',
-      displayName: 'Machine Learning',
-      lmsLink: '//localhost:18000/courses/course-v1:MachineLearning+123+2023/jump_to/block-v1:MachineLearning+123+2023+type@course+block@course',
-      number: '123',
-      org: 'LSE',
-      rerunLink: '/course_rerun/course-v1:MachineLearning+123+2023',
-      run: '2023',
-      url: '/course/course-v1:MachineLearning+123+2023',
-    },
-    {
-      courseKey: 'course-v1:Design+123+e.g.2025',
-      displayName: 'Design',
-      lmsLink: '//localhost:18000/courses/course-v1:Design+123+e.g.2025/jump_to/block-v1:Design+123+e.g.2025+type@course+block@course',
-      number: '123',
-      org: 'University of Cape Town',
-      rerunLink: '/course_rerun/course-v1:Design+123+e.g.2025',
-      run: 'e.g.2025',
-      url: '/course/course-v1:Design+123+e.g.2025',
-    },
-  ],
-  courses: [
-    {
-      courseKey: 'course-v1:HarvardX+123+2023',
-      displayName: 'Managing Risk in the Information Age',
-      lmsLink: '//localhost:18000/courses/course-v1:HarvardX+123+2023/jump_to/block-v1:HarvardX+123+2023+type@course+block@course',
-      number: '123',
-      org: 'HarvardX',
-      rerunLink: '/course_rerun/course-v1:HarvardX+123+2023',
-      run: '2023',
-      url: '/course/course-v1:HarvardX+123+2023',
-    },
-    {
-      courseKey: 'org.0/course_0/Run_0',
-      displayName: 'Run 0',
-      lmsLink: null,
-      number: 'course_0',
-      org: 'org.0',
-      rerunLink: null,
-      run: 'Run_0',
-      url: null,
-    },
-  ],
-  inProcessCourseActions: [],
+  count: 5,
+  next: null,
+  previous: null,
+  numPages: 2,
+  results: {
+    archivedCourses: [
+      {
+        courseKey: 'course-v1:MachineLearning+123+2023',
+        displayName: 'Machine Learning',
+        lmsLink: '//localhost:18000/courses/course-v1:MachineLearning+123+2023/jump_to/block-v1:MachineLearning+123+2023+type@course+block@course',
+        number: '123',
+        org: 'LSE',
+        rerunLink: '/course_rerun/course-v1:MachineLearning+123+2023',
+        run: '2023',
+        url: '/course/course-v1:MachineLearning+123+2023',
+      },
+      {
+        courseKey: 'course-v1:Design+123+e.g.2025',
+        displayName: 'Design',
+        lmsLink: '//localhost:18000/courses/course-v1:Design+123+e.g.2025/jump_to/block-v1:Design+123+e.g.2025+type@course+block@course',
+        number: '123',
+        org: 'University of Cape Town',
+        rerunLink: '/course_rerun/course-v1:Design+123+e.g.2025',
+        run: 'e.g.2025',
+        url: '/course/course-v1:Design+123+e.g.2025',
+      },
+    ],
```

Comment:
In the V2 API we're using the key `courses` for archived/active courses, so we're not sending them in the same response. Is this Mock response up to date?

Reply:
I've removed them from the mock